B/bsdiff_classic/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "bsdiff_classic"
-version = v"4.3.0"
```

Comment:
Does this need to be updated or should I leave it as 4.3.0?

Reply:
You do need to change the version number if compat bounds change (and you changed also the julia one below)